### PR TITLE
[Re-Sprite] New Dectetive Sprites + New Skirt, Black Jumpsuit and Black Jacket!

### PR DESCRIPTION
**Pull request by Dapple27**
## What Does This PR Do

Re-sprites detective stuff including forensics gloves and black fedora, also, adds a detective skirt, black jumpsuit and jacket.

## Why It's Good For The Game

The entirely sec department were re-sprited, except det's stuff, and the old sprites are very constranting between new sprites.

## Images of changes
detdrask
detdrask1
detkidan
detkidan1
detvox
detvox2
detgrey
detgrey2
det2
det1
detobj
Screenshot_2

## Changelog
:cl:
add: New detective's skirt
add: New black forensics jumpsuit and jacket, available in detdrobe
imageadd: Detective's skirt, forensics jumpsuit and jacket sprites
imageadd: Sprites sheets of det's Stuff (gloves, suits, uniforms) for Kidan, Grey, Drask and Vox 
imageadd: Resprites det's stuff 
/:cl:

### CODE REVIEW

**Comment by Carthusia** [comment]
Sad plasmamen dont get any new bits, they always are left out due to the suits

**Comment by Henri215** [comment]
Drask hands are supposed to look different from each other?

**Comment by Dapple27** [comment]
> Sad plasmamen dont get any new bits, they always are left out due to the suits

there is a det enviosuit for plasmamen?, i could do it later, for now, i'm tired of making this, i hate doing the same uniform/suits three times for non-humanoid species :skull: 

~~delete non-humanoids pls~~

**Comment by Dapple27** [comment]
> Drask hands do look different from each other?! How have i only noticed this now?!

well, i only used the preview that i made to arm body like legos, those are their real hands apparently, kinda weird, but yes.
image

**Comment by Carthusia** [comment]
> > Drask hands do look different from each other?! How have i only noticed this now?!
> 
> well, i only used the preview that i made to arm body like legos, those are their real hands apparently, kinda weird, but yes. image

My god i never noticed, all i can hear now is a drask as that guy in scary movie 2:

"Grab my hand!"
(Holds out gross, slimey deformed hand)
{NO USE YOUR OTHER HAND!}
"No, this is mah strong hand!"
{AGHHHHHHHHHHHHHhhhhhhhhhhhhhhhh....}
(Falls to his death)

**Review by Sirryan2002 — APPROVED** [review]
small thing to do before merge

File: code/modules/clothing/head/misc_hats.dm [hunk 1]
```diff
@@ -274,6 +274,11 @@
 	item_state = "fedora"
 	desc = "A great hat ruined by being within fifty yards of you."
 	actions_types = list(/datum/action/item_action/tip_fedora)
+	sprite_sheets = list(
+	"Vox" = 'icons/mob/clothing/species/vox/head.dmi',
+	"Grey" = 'icons/mob/clothing/species/grey/head.dmi'
```
[the remark this answers is not in the document]
Comment (Dapple27) [reply code/modules/clothing/head/misc_hats.dm]: done

**Review by S34NW — APPROVED** [review]
Please deconflict

**Pull request merged**